webdriver-spec.html

```diff
@@ -1262,6 +1263,15 @@
  </tr>
 
  <tr>
+  <td><dfn>element intercepting click</dfn>
+  <td>400
+  <td><code>element intercepting click</code>
```

Comment:
"element click intercepted". The other errors tend to be in the past tense, not the present tense.